development/webpack/webpack.desktop.config.js

```
@@ -21,7 +21,9 @@ module.exports = ({
   const commonDesktopConfig = {
     externals: {
       // Exclude the entire BLE transport package to prevent Node.js modules from leaking to renderer
-      '@onekeyfe/hd-transport-electron': 'commonjs @onekeyfe/hd-transport-electron',
+      '@onekeyfe/hd-transport-electron': 'commonjs @onekeyfe/hd-transport-electron',      
```

ESLint

[error] Replace `·'commonjs·@onekeyfe/hd-transport-electron',······` with `⏎········'commonjs·@onekeyfe/hd-transport-electron',`

(prettier/prettier)